core/App/components/record/RecordField.tsx

```diff
@@ -65,13 +66,18 @@ const RecordField: React.FC<RecordFieldProps> = ({
         </Text>
       )}
       <View style={styles.valueContainer}>
-        {fieldValue ? (
+        {fieldValue ? 
+        (
           fieldValue(field)
         ) : (
           <>
             <View style={styles.valueText}>
               <Text style={styles.text} testID={testIdWithKey('AttributeValue')}>
-                {shown ? (field as Attribute).value : Array(10).fill('\u2022').join('')}
+                {shown ? 
+                dateAttributes.includes(field.name!) ?
+                new Date((field).value * 1000).toLocaleDateString('en-CA', dateFormatOptions) :
+                (field as Attribute).value 
+                : Array(10).fill('\u2022').join('')}
```

Comment:
I realize this is not your code--but do you know what this is doing here @ryankoch13? As this appears to be hardcoded in some form to 2022, which is soon going to be over

Reply:
it's for unicode character 2022, it's the little star bullet thing they use